v2/coord/query/query.go

```diff
@@ -130,14 +130,18 @@ func (q *Query[K, N]) Advance(ctx context.Context, ev QueryEvent) QueryState {
 
 	switch tev := ev.(type) {
 	case *EventQueryCancel:
+		span.SetAttributes(tele.AttrEvent("EventQueryCancel"))
```

Comment:
could have something like:

```go
span.SetAttributes(tele.AttrEvent(fmt.Sprintf("%T", ev)))
```

at the top of this method. `fmt` is already imported.

Reply:
or add it right to L122

Reply:
I'm wary of adding allocations for tracing since they will affect performance